libraries/ms-helper/src/x86/adc_periodic_reader.c

```diff
@@ -0,0 +1,82 @@
+#include "adc_periodic_reader.h"
+#include "adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "gpio_mcu.h"
+#include "interrupt.h"
+#include "log.h"
+#include "misc.h"
+#include "soft_timer.h"
+#include "status.h"
+
+#define TIMER_INTERVAL 50

```

Comment:
This should probably be an init option

Reply:
Hmm I could change it to be an init option, would it affect the functionality a lot? My tests rely on this exact interval of 50ms

Reply:
you can make it a constant in the test that you pass to the periodic reader. In the case of steering tests it should be a "public" define that the tests have access to and use I guess. I don't think it should be a big deal to change, but would make things cleaner if they end up being changed later